lib/internal/loader/DefaultResolve.js

```diff
@@ -39,14 +43,22 @@ function search(target, base) {
   }
 }
 
-const extensionFormatMap = {
+const extensionFormatStd = {
   __proto__: null,
   '.mjs': 'esm',
   '.json': 'json',
   '.node': 'addon',
   '.js': 'commonjs'
 };
```

Comment:
☝️ still not a fan of using short forms for most names but not `commonjs`. We have ECMAScript module -> `esm`, JavaScript Object Notation -> `json`, Common JS -> should be `cjs`.

Reply:
tbh I have no idea why I made this change and if it had been brought up before my PR landed I would have most likely changed it back, kinda feelin' guilty about this one.

Reply:
I would also like to see it changed back to `"cjs"`